build/SetEnv.tasks

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Because this file appears to be used only in `SetEnv.targets`, I don't think it passes the [reused abstraction rule](http://codemanship.co.uk/parlezuml/blog/?postid=934) yet. It would be easier to understand this code if the SetEnv task was defined in the `SetEnv.targets` file.